### PR TITLE
feat(es_extended/client/blips): add Blip class

### PR DESCRIPTION
**Pull request by btwlouis**
This pull request introduces a new `Blip` class to simplify the management of blips. The class encapsulates all functionalities related to blip creation, updates, and deletion, providing an easy-to-use interface for developers.

# Benefits
- Simplifies blip management
- Improves readability and maintainability of the codebase
- Reduces code repetition by encapsulating blip operations into reusable methods

# Example:

```lua
CreateThread(function()
    local blip = ESX.Blip:new({
        coords = vec3(202.50, -841.45, 30.67),
        sprite = 23,
        colour = 2,
        label = "Test Blip",
        scale = 1.5,
        display = 4,
        shortRange = true
    })

    Wait(1000)

    blip:setColour(3)

    Wait(1000)

    blip:setLabel("Test Blip 2")

    Wait(1000)

    blip:setScale(2.0)

    Wait(1000)

    blip:setDisplay(2)

    Wait(1000)

    blip:setShortRange(false)

    Wait(1000)

    blip:setCoords(vec3(202.50, -845.45, 30.67))
end)
```

# Impact on Existing Code
This addition is fully backward-compatible. It does not alter any existing functionality within the framework but provides a modular approach for future blip-related operations.

# Next Steps
- Extend the `Blip` class with more options to modfiy a blip

### CODE REVIEW

File: [core]/es_extended/imports.lua [hunk 1]
```diff
@@ -29,7 +29,7 @@ if not IsDuplicityVersion() then -- Only register this event for the client
 
         local file = LoadResourceFile("es_extended", path)
         if file then
-            local fn, err = load(file, ('@@es_extended/%s'):format(path))
```
Comment (Mycroft-Studios) [inline [core]/es_extended/imports.lua:32]: This should remain the same

**Review by Arctos2win — CHANGES_REQUESTED** [review]
put the whole logic into the imports file no need to have internal functions on this one.

**Comment by Kenshiin13** [comment]
@Arctos2win

**Comment by Kenshiin13** [comment]
Hey @btwlouis,

Thank you for your contribution! After careful consideration and discussion with the team, we've decided that this feature isn't needed at the moment. We really appreciate the effort you put into it and hope you'll continue contributing in the future. Thank you again!

**Pull request closed without merging**